src/Block/Identifying.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Wearesho\Bobra\Ubki\Block;
+
+use Wearesho\Bobra\Ubki;
+
+/**
+ * Class Identifying
+ * @package Wearesho\Bobra\Ubki\Block
+ */
+class Identifying extends Ubki\Block
+{
+    protected $credential;
+    
+    public function __construct($credential)
```

Comment:
WIll be refactored after Credential element implement

Reply:
Where is todo?